libs/cart/src/injection-tokens/payment-method-map.ts

```diff
@@ -0,0 +1,11 @@
+import { InjectionToken } from '@angular/core';
+
+/**
+ * An injection token for a mapping from the platform-specific cart payment method
+ * to a user-defined platform-agnostic payment ID.
+ * It should be an object whose keys are the cart payment's method.
+ * Defaults to an empty object.
+ */
+export const DaffCartPaymentMethodMap = new InjectionToken<Object>('DaffCartPaymentMethodMap', {
```

Comment:
This should be `DaffCartPaymentMethodIdMap`, right? To be a little more clear to the user

Reply:
sounds good, changed